### ISSUE TITLE
`hugo serve` hangs on shutdown if a tab is open in Firefox

### ISSUE THREAD

**Issue by hkoosha** [gohugoio/hugo#12686]
### What version of Hugo are you using (`hugo version`)?

<pre>
$ hugo version
hugo v0.129.0+extended linux/amd64 BuildDate=unknown
</pre>

### Does this issue reproduce with the latest release?
Yes

### The issue

When running Hugo's builtin web-server and visiting `localhost:1313` with Firefox, then _after_ clicking any link and navigating through the website, apparently FF keeps a connection to Hugo open. Then, switching back to Hugo and trying to shut it down via `CTRL+C`, it takes a very long time until Hugo finally shuts down (almost forever). Other browsers do not do this.

One solution is to chase every Hugo tab in FF and close them, then it shuts down quickly; but it's disrupting a bit.

Does it make sense to ignore open connections on `CTRL+c`?

**Comment by bep** [comment]
>Does it make sense to ignore open connections on CTRL+c?

Yes it does. I vaguely remember a similar issue, but not the details ...
